### PR TITLE
Clarify binary operator output label sets

### PR DESCRIPTION
**Pull request by juliusv**
Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

File: docs/querying/operators.md [hunk 1]
```diff
@@ -69,7 +70,7 @@ modifier is provided, vector elements that would be dropped instead have the val
 applied to matching entries. Vector elements for which the expression is not
 true or which do not find a match on the other side of the expression get
 dropped from the result, while the others are propagated into a result vector
-with their original (left-hand side) metric names and label values.
+with the matching labels becoming the output label set.
 If the `bool` modifier is provided, vector elements that would have been
 dropped instead have the value `0` and vector elements that would be kept have
 the value `1` with the left-hand side label values.
```
Comment (brian-brazil) [inline docs/querying/operators.md]: Hmm, we say LHS labels here.

File: docs/querying/operators.md [hunk 2]
```diff
@@ -69,7 +70,7 @@ modifier is provided, vector elements that would be dropped instead have the val
 applied to matching entries. Vector elements for which the expression is not
 true or which do not find a match on the other side of the expression get
 dropped from the result, while the others are propagated into a result vector
-with their original (left-hand side) metric names and label values.
+with the matching labels becoming the output label set.
```
Comment (brian-brazil) [inline docs/querying/operators.md]: grouping labels

**Comment by juliusv** [comment]
@brian-brazil Changed "matching labels" to "grouping labels" in both instances and changed the `bool` case description as well.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**